product-service/handlers/createProduct.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+import { executeTransaction } from "../utils/db.js";
+import {handleRequest} from "../utils/request.js";
+
+export const create = async (event) => {
+    let product
+    try {
+        product = JSON.parse(event.body)
+        if (!product.manufacturer || !product.model || !product.price || !product.count) {
+            return {
+                statusCode: 400,
+                body: "Incorrect product model. Missing manufacturer, model, price or count"
+            }
+        }
+    } catch (error) {
+        return {
+            statusCode: 400,
+            body: "Cannot parse request body",
+        };
+    }
+
+    return executeTransaction(async (client) => {
+        const {manufacturer, model, img, price, count} = product
+        const queryProduct = `insert into products (manufacturer, model, img, price) values
```

Comment:
[Parameterized query](https://node-postgres.com/features/queries#parameterized-query) should be used for protection from sql injections.

Reply:
Fixed